tests/nn/data_parallel/test_fsdp_regnet.py

```diff
@@ -190,6 +310,15 @@ def _test_func(
         # Move tensors to CPU to compare numerics.
         for k, v in fsdp_state.items():
             fsdp_state[k] = v.cpu()
+        # Enable for debugging.
```

Comment:
nit: Should we remove this for long running tests?  I guess you are still debugging the issue so it makes sense to have it. only concern here is that it adds to the execution time.

Reply:
it is disabled with "if False". I will clarify it.